libclc/cmake/modules/AddLibclc.cmake

```diff
@@ -456,7 +456,7 @@ function(add_libclc_builtin_set)
             "${LIBCLC_LIBRARY_OUTPUT_INTDIR}/remangled-${long_width}-${signedness}_char.${obj_suffix_mangled}" )
         add_custom_command( OUTPUT "${builtins_remangle_path}"
           COMMAND ${CMAKE_COMMAND} -E make_directory ${LIBCLC_LIBRARY_OUTPUT_INTDIR}
-          COMMAND libclc::libclc-remangler
+          COMMAND ${LIBCLC_REMANGLER_PATH}
```

Comment:
Perhaps the variable should be named `libclc-remangler_exe` like the other tools: `clang_exe`, `llvm-link_exe`?

Reply:
Same thing as my previous comment above, but yes:

> Yes it should've but I have decided to not do all the cross compiling logic manually, instead using `setup_host_tool`.